app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.utils.JwtUtil;
+import io.jsonwebtoken.Claims;
+import org.springframework.stereotype.Service;
+
+/**
+ * 로그인 로직을 수행합니다.
+ */
+@Service
+public class AuthenticationService {
+
+    JwtUtil jwtUtil;
+
+    public AuthenticationService(JwtUtil jwtUtil) {
+        this.jwtUtil = jwtUtil;
+    }
+
+    /**
+     * 사용자 정보로 로그인하여 JWT 를 리턴합니다.
+     * @return 생성된 JWT
+     */
+    public String login() {
+        return jwtUtil.encode(1L);
+    }
+
+    /**
+     * 발행된 JWT 를 복호화 합니다.
```

Comment:
메소드 시그니처를 보면 `JWT`를 굳이 드러낼 필요가 없어 보여요. JWT를 사용한다는 것은 메소드 본문을 읽어야 알 수 있는 정보입니다.
```suggestion
     * 주어진 엑세스 토큰에서 사용자 아이디를 복호화해 리턴합니다.
```

Reply:
아 JWT가 발급 된 것은 해당 메서드와는 별개이기 때문에, 본 javaDoc 에선 드러낼 필요가 없는건가요?

Reply:
네. 정보 은닉 원칙을 떠올려 보세요.